proactive-jupyter-kernel/kernel.py

```diff
@@ -204,6 +204,14 @@ def __trigger_pragma__(self, pragma_info):
             return self.__delete_task__
         elif pragma_info['trigger'] == 'list_submitted_jobs':
             return self.__list_submitted_jobs__
+        elif pragma_info['trigger'] == 'get_target_env_nodesources':
```

Comment:
Why having explicit pragmas for `get_target_env_nodesources`, `get_target_env_hosts` and `get_target_env_tokens` instead of having a parametrised `get_target_envs` instead?